README.md

```diff
@@ -30,39 +30,87 @@ Somewhere else on your host system, create a local clone of the Ceph git
 repository. Replace `<ceph-repository>` with the remote git repo you want to
 clone from, e.g. `https://github.com/ceph/ceph.git`:
 
-    # cd <workdir>
-    # git clone <ceph-repository>
-    # cd ceph
+    $ cd <workdir>
+    $ git clone <ceph-repository>
+    $ cd ceph
 
 Now switch or create your development branch using `git checkout` or `git
 branch`.
-    
+
 ### Starting the Container and building Ceph
 
 Now start up the container, by mounting the local git clone directory as
 `/ceph`:
 
-    # docker run -it -v $PWD:/ceph --net=host ceph-dev-docker /bin/bash
+    $ docker run -itd \
```

Comment:
Other solution might be to add a note at the top of the document that all docker commands need to be executed as root or the current user must join the docker group (don't knwo if this is the correct group name).

Reply:
done.